pkg/api/annotations/annotations.go

```diff
@@ -16,8 +16,35 @@ limitations under the License.
 
 package annotations
 
-const kubectlPrefix = "kubectl.kubernetes.io/"
+const (
```

Comment:
I don't know why there is a package pkg/api/annotation. I think we should get rid of the this package, and copy the annotations to pkg/api/annotation_key_constants.go and pkg/api/v1/annotation_key_constants.go, like other api.